src/components/buttons/AddRemoveButton.tsx

```diff
@@ -1,36 +0,0 @@
-import React from "react";
-
-interface AddRemoveButtonProps {
-  children: string;
-  buttonType: "add" | "remove";
-  onClick: () => void;
-}
-
-export default function AddRemoveButton({
-  children,
-  buttonType,
-  onClick,
-}: AddRemoveButtonProps) {
-  let bgColor;
-
-  switch (buttonType) {
-    case "add":
-      bgColor = "bg-emerald-800 shadow-emerald-800/50 hover:bg-emerald-600";
-      break;
-    case "remove":
-      bgColor = "bg-deepRed shadow-deepRed/50 hover:bg-red-600";
-      break;
-  }
-
-  return (
-    <button
-      type="button"
-      className={`shadow ${bgColor} mb-1 min-w-full self-end rounded py-1 text-xs font-semibold text-white`}
-      onClick={() => onClick()}
```

Comment:
Since `onClick` type does not contain any parameters (which suggests that you never want to pass any parameters), you can just shorten this to `onClick={onClick}`